flutter/lib/src/default_integrations.dart

```diff
@@ -325,9 +325,9 @@ class _LoadContextsIntegrationEventProcessor extends EventProcessor {
   }
 }
 
-/// Enables Sentry's native SDKs (Android and iOS)
-class NativeSdkIntegration extends Integration<SentryFlutterOptions> {
-  NativeSdkIntegration(this._channel);
+/// Initializes Sentry's native SDKs (Android and iOS) with options.
+class InitNativeSdkIntegration extends Integration<SentryFlutterOptions> {
```

Comment:
I'd rather keep the same name, since we already report the integration as `nativeSdkIntegration` otherwise its a breaking change and we can't check the usage of it.